### ISSUE TITLE
Possible update to 1.7.10

### ISSUE THREAD

**Issue by GlorfindeI** [LatvianModder/Project-EX#80]
I love this mod so much and I when I dicovered that I could not play it with my friends on our 1.7.10 server I was very sad. I was wondering if you will ever make a version of this mod for 1.17.10 or for any other older versions of Minecraft. I know that there are a lot of people who would love to see this mod playable on older versions. 

Even if you just dismiss this without any thought to it, I want to thank you for creating this wonderful mod.

**Comment by mhmyesman** [comment]
I completely agree, I think it would be a shame if this mod was constricted to only 1.12. Making the mod 1.7.10 compatible would be a lot of work, but I think that it would be worth the effort. 

Thank you again for considering.

**Comment by MRtecno98** [comment]
I asked the same question some time ago. I agree with you it would be absolutely amazing to have this mod for older versions too.

**Comment by BlueAgent** [comment]
Might be difficult without changes on project E's end as well.
IIRC in 1.7.10, the transmutation table's EMC is stored as an integer instead of a long.

**Comment by zeebrandonite** [comment]
If there is one thing I would want, it's this mod in 1.7.10 i.e. there are so many items I can't transmute because of the max EMC value being an int and not a long
